docs/language/learn-ql/python/introduce-libraries-python.rst

```diff
@@ -1,16 +1,18 @@
-Introducing the CodeQL libraries for Python
-===========================================
+CodeQL library for Python
+=========================
 
-There is an extensive library for analyzing CodeQL databases extracted from Python projects. The classes in this library present the data from a database in an object-oriented form and provide abstractions and predicates to help you with common analysis tasks.  The library is implemented as a set of QL modules, that is, files with the extension ``.qll``. The module ``python.qll`` imports all the core Python library modules, so you can include the complete library by beginning your query with:
+This is an overview of the extensive library you use to analyze databases generated from Python code bases. Using this library makes it easier for you to write queries.
 
-.. code-block:: ql
+About the CodeQL library for Python
+-----------------------------------
 
-   import python
+The CodeQL library for each programming language uses classes with abstractions and predicates to present data in an object-oriented form. This abstraction makes it easier for you to write queries.
```

Comment:
Perhaps remove the final sentence? That avoids duplicating "makes it easier for you to write queries."

Reply:
```suggestion
The CodeQL library for each programming language uses classes with abstractions and predicates to present data in an object-oriented form.
```